2023-05-30T11:20:19|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\Users\ZHao\.conda\envs\scopefoundry\lib\site-packages\ScopeFoundry\measurement.py", line 25, in run
    self.measurement._thread_run()
  File "C:\Users\ZHao\.conda\envs\scopefoundry\lib\site-packages\ScopeFoundry\measurement.py", line 179, in _thread_run
    self.run()
  File "C:\Users\Public\quantum_sensing\quantum_sensing\fancy_microscope\thorcam_capture.py", line 254, in run
    sdk = TLCameraSDK()
  File "C:\Users\ZHao\.conda\envs\scopefoundry\lib\site-packages\thorlabs_tsi_sdk\tl_camera.py", line 172, in __init__
    raise TLCameraError("TLCameraSDK is already in use. Please dispose of the current instance before"
thorlabs_tsi_sdk.tl_camera.TLCameraError: TLCameraSDK is already in use. Please dispose of the current instance before trying to create another

2023-05-30T11:24:26|CRITICAL|root|Unhandled exception:Traceback (most recent call last):
  File "C:\Users\ZHao\.conda\envs\scopefoundry\lib\site-packages\ScopeFoundry\helper_funcs.py", line 93, in eventFilter
    def eventFilter(self, obj, event):
KeyboardInterrupt

